### PR TITLE
Switch to `@` for Attributes

### PR DESCRIPTION
**Pull request by mehmetoguzderin**
This PR implements 2022 January 19 consensus to use `@` for attributes as reasoned in https://github.com/gpuweb/gpuweb/issues/2243 issue.

***
<a href="https://pr-preview.s3.amazonaws.com/mehmetoguzderin/gpuweb/pull/2517.html" title="Last updated on Jan 19, 2022, 2:22 AM UTC (8d33e28)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/gpuweb/gpuweb/2517/4349dce...mehmetoguzderin:8d33e28.html" title="Last updated on Jan 19, 2022, 2:22 AM UTC (8d33e28)">Diff</a>

### CODE REVIEW

**Review by dneto0 — CHANGES_REQUESTED** [review]
It looks good to me.
But I just landed #2503  and so there are conflicts to be resolved.

**Comment by mehmetoguzderin** [comment]
@dneto0 tried to resolve, hope I didn't mess it!

File: wgsl/index.bs [hunk 1]
```diff
@@ -1859,7 +1854,7 @@ rounded to the next multiple of the structure's alignment:
         u: f32;                                    // offset(0)   align(4)  size(4)
         v: f32;                                    // offset(4)   align(4)  size(4)
         w: vec2<f32>;                              // offset(8)   align(8)  size(8)
-        [[size(16)]] x: f32;                       // offset(16)  align(4)  size(16)
+        @size(16) x: f32;                       // offset(16)  align(4)  size(16)
```
Comment (dneto0) [inline wgsl/index.bs:1857]: We can fix the comment placement later, when we make everything pretty.

**Review by dneto0 — APPROVED** [review]
Thanks!

**Comment by mehmetoguzderin** [comment]
@dneto0 thanks for the merge!

**Pull request merged**